docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -722,7 +722,7 @@ testcafe ie my-tests --page-load-timeout 0
 
 Specifies the test execution speed.
 
-Tests are run at the maximum speed by default. You can use this option
+Tests run at the maximum speed by default. You can use this option
```

Comment:
- Why do you use "by default"?
- "You can use this option to slow the test down" -> You can use this option to adjust/change test speed

Reply:
You're right, this sentence violates guidelines.

Does this work: 

> If this parameter is not specified, tests run at the maximum speed. You can use this option to decrease test speed.

Reply:
Ok